templates/group-tabs.build.hbs

```diff
@@ -0,0 +1,6 @@
+<section class="group-tabs-holder">
+  <input type="radio" name="group-tabs" checked="checked" id="group-tabs-0" value="" />
+  <label for="group-tabs-0">Direct messages</label>
+  <input type="radio" name="group-tabs" id="group-tabs-1" value="1" />
+  <label for="group-tabs-1">Public channels</label>
+</section>
```

Comment:
https://jsfiddle.net/tonytlwu/rs1ebv8j/ is ready to go. You could swap it in now (though I'm wondering whether the CSS part should be added to a `fliplet-api` package. Maybe in `bootstrap-css`.